src/main/java/kostka/moviecatalog/entity/counters/SynchronizedCounter.java

```diff
@@ -0,0 +1,25 @@
+package kostka.moviecatalog.entity.counters;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
```

Comment:
Proc je tohle JPA entita?

Reply:
Jak jsem psal, zkoušel jsem víc možností jak to udělat. Tajze pozůstatek :-)